ClientCollection/Me/index.ts

```diff
@@ -14,21 +14,20 @@ export class Me extends rest.Collection<gracely.Error> {
 	) {
 		super(client)
 	}
-	async login(credentials: User.Credentials): Promise<User.Key | gracely.Error> {
+	async login(credentials: User.Credentials | userwidgets.User.Key): Promise<User.Key | gracely.Error> {
 		let result: gracely.Error | User.Key
-		if (credentials.password == undefined)
-			result = gracely.client.malformedContent("password", "string", "Password is required for login.")
-		else {
-			const token = await this.client.get<string>(`${this.configuration.pathPrefix}/me`, {
-				authorization: User.Credentials.toBasic({ user: credentials.user, password: credentials.password }),
-			})
-			result = gracely.Error.is(token)
-				? token
-				: (await User.Key.Verifier.create(this.configuration.publicKey).verify(token)) ??
-				  gracely.client.unauthorized("Failed to verify token.")
-			if (!gracely.Error.is(result))
-				this.keySetter(result.token)
-		}
+		const token = await this.client.get<string>(`${this.configuration.pathPrefix}/me`, {
+			authorization:
+				"token" in credentials
+					? `Bearer ${credentials.token}`
```

Comment:
Manually adding this header should not be necessary. The client have received the token from the previous login and already knows about it ( https://github.com/userwidgets/model/blob/master/ClientCollection/index.ts#L47). If we have a token just exclude the header. Only do this if that does not work.